models/mrsegmentator/config/low_memory.yml

```diff
@@ -0,0 +1,34 @@
+general:
```

Comment:
I suggest we call this workflow `fast.yml`, what do you think?

Reply:
That sounds great!